state.go

```diff
@@ -532,6 +533,7 @@ func (o *OverflowState) InitializeContracts(ctx context.Context) *OverflowState
 
 // GetAccount takes the account name  and returns the state of that account on the given network.
 func (o *OverflowState) GetAccount(ctx context.Context, key string) (*flow.Account, error) {
+	ctx = logging.InjectLogField(ctx, "account_name", key)
```

Comment:
can we add address here as a seperate logging field, i belive it can have value to have both here. the name is totally local to your configuration while the address could be valuable remotely aswell.